jx_sqlite/query_table.py

```diff
@@ -87,8 +87,8 @@ def query(self, query):
             Log.error("Expecting table, or some nested table")
         frum, query['from'] = query['from'], self
         schema = self.sf.tables["."].schema
-        query = QueryOp.wrap(query, schema)
-
+        if not query.groupby:
+            query = QueryOp.wrap(query, schema)                        
```

Comment:
line 121 ` elif query.edges or any(a != "none" for a in listwrap(query.select).aggregate):`
expects query to be already normalised because it's a dict type.
Error: string indices must be integers.
So query is need to be normalized before the if block.